tests/e2e/test_function.py

```diff
@@ -1,16 +1,25 @@
 import os
 
+import pytest
 from dotenv import load_dotenv
 
 load_dotenv()
 
+# To run this tests we need to have Akamai integration set up
+# Only run when Akamai property is configured
+# When running from terminal or with python -m pytest function tests will
+# be skipped
+# To enable tests, remove skip annotation or run with CMD/IDE
```

Comment:
Should we add in Pull Request Template.. saying "If changes are made in functions module please attach a screenshot of function tests run (as they are skipped normally)" ?? 
Basically just to make any changes being done on functions are being tested as well ?

Reply:
Great idea!

Reply:
@koshy-thomas done done